device/jsonld-contexts/device.jsonld

```diff
@@ -9,6 +9,13 @@
         "internalHumidity": "https://smartdatamodels.org/dataModel.Device/internalHumidity",
         "internalTemperature": "https://smartdatamodels.org/dataModel.Device/internalTemperature",
         "loraSNR": "https://smartdatamodels.org/dataModel.Device/loraSNR",
-        "softwareVersion": "https://smartdatamodels.org/dataModel.Device/softwareVersion"
+        "softwareVersion": "https://smartdatamodels.org/dataModel.Device/softwareVersion",
+        "deviceDescription": "https://smartdatamodels.org/dataModel.Device/deviceDescription",
+        "devEUI": "https://smartdatamodels.org/dataModel.Device/devEUI",
+        "appEUI": "https://smartdatamodels.org/dataModel.Device/appEUI",
+        "deviceIsDeleted": "https://smartdatamodels.org/dataModel.Device/isDeleted",
+        "deviceNetworks": "https://smartdatamodels.org/dataModel.Device/deviceNetworks",
+        "networkIsEnabled": "https://smartdatamodels.org/dataModel.Device/networkIsEnabled",
+        "networkName": "https://smartdatamodels.org/dataModel.Device/networkName"

```

Comment:
add them in alphabetical order, easier to read later

Reply:
Done